### PR TITLE
Update EYOC help doc and add new pages

### PR DESCRIPTION
**Pull request by bondydaa**
Creating new pages and updating content for the Expensify Your Own Conference (EYOC) pages.

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/311975
PROPOSAL: 

### Tests

- [x] Verify that no errors appear in the JS console

### QA Steps

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] If we are not using the full Onyx data that we loaded, I've added the proper selector in order to ensure the component only re-renders when the data it is using changes
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

the EYOC main page:
eyoc

Updates TOC with new links:
new_pages_toc

For Admins page:
chat_admins

For Attendees page:
chat_attendees

For Speakers page:
chat_speakers

</details>

<details>
<summary>Mobile Web - Chrome</summary>

</details>

<details>
<summary>Mobile Web - Safari</summary>

</details>

<details>
<summary>Desktop</summary>

</details>

<details>
<summary>iOS</summary>

</details>

<details>
<summary>Android</summary>

</details>

### CODE REVIEW

**Comment by bondydaa** [comment]
Mostly ready for review, waiting on some updated content from the marketing team but let me know if there is anything glaringly wrong with the format of the files or something.

File: docs/articles/other/Expensify-Chat-For-Conference-Attendees.md
Comment (dangrous) [inline docs/articles/other/Expensify-Chat-For-Conference-Attendees.md:26]:
```suggestion
**Don't:**
```

File: docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md
Comment (dangrous) [inline docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md:11]: Does Jekyll cut down on extra spaces? This doc is formatted pretty differently to the other one, with a lot more whitespace before/after section headers, etc.

**Review by dangrous — DISMISSED** [review]
One question and one very minor comment, but nothing looks weird or out of place! will hold on a final review for the final copy of course

File: docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md
Comment (bondydaa) [reply docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md:11]: that's a good question, i can't remember. I'll spin these up locally and get screenshots!

**Comment by bondydaa** [comment]
okay got this updated and tested locally finally. 

I believe we're trying to get this out pretty quickly.

**Comment by bondydaa** [comment]
@allroundexperts could you just do the checklist? not sure there is much you can actually test though since this is the help doc site.

**Review by allroundexperts — APPROVED** [review]
Testing well!

**Comment by bondydaa** [comment]
ah looks like Daniel is OOO sick today. Going to see if I can get another set of eyes to approve and merge. going to dismiss his review since I fixed the one thing he called out.

**Comment by bondydaa** [comment]
oh neat i guess @aldo-expensify you're the lucky duck 😂

File: docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md
Comment (aldo-expensify) [inline docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md:37]:
```suggestion
**Protip**: Your Expensify account manager can complete this step with you. Chat them in #admins on [new.expensify.com](https://new.expensify.com) to coordinate!
```
NAB

NAB: Also, is "Chat them" phrased like that on purpose instead of "Chat with them"?

File: docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md
Comment (aldo-expensify) [inline docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md:78]:
```suggestion
- We’d also recommend posting your updates on individual messages so that if someone has a question about a certain item they can ask in a thread pertaining to that topic, rather than in one consolidated block.
```

NAB: You can do many new lines in a single message and it won't allow you to create separate threads

File: docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md
Comment (aldo-expensify) [inline docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md:87]: These pages are not ready yet, right?
image

Comment (aldo-expensify) [reply docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md:87]: Out of the scope of this PR, but the not found page looks pretty broken

**Review by aldo-expensify — APPROVED** [review]
Left some NABS

File: docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md
Comment (bondydaa) [reply docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md:37]: This was directly from the doc. https://docs.google.com/document/d/15OHoH9i9bipRTsEBi5Pud0m3TK3-i_jhv6emBvRrZqM/edit

cc @LLPeckham

File: docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md
Comment (bondydaa) [reply docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md:87]: yeah we are adding these in this PR. 

We could probably use relative URLs to make life easier but not a concern atm.

**Comment by bondydaa** [comment]
I'm going to merge this so the content is live and we can test. 

we can easily make copy updates as needed.

**Comment by aldo-expensify** [comment]
Works for me!

**Comment by dangrous** [comment]
Thanks for the push on this! Sorry I was out for the end bit

**Pull request merged**
